src/utils/configured-graphql-client.ts

```diff
@@ -0,0 +1,21 @@
+import {GraphQLClient} from 'graphql-request'
+import config from '../lib/config'
+
+export const setAuthorizationHeaderForClient = (
+  client: GraphQLClient,
+  authToken: string | undefined,
+) => {
+  if (authToken) {
+    const authorizationHeader = `Bearer ${authToken}`
+    client.setHeader('authorization', authorizationHeader)
+  }
+}
+
+export const getGraphQLClient = (token?: string) => {
```

Comment:
I'm glad you tackled this refactor 🙌 I almost scratched this itch the other day when setting up the boilerplate for a graphql request.

Reply:
changed it to accept a token like you suggested!